fix PR template (#4339)
is-a-dev/register: .github/PULL_REQUEST_TEMPLATE.md

<!-- To make our job easier, please spend time to review your application before submitting. -->

> <!-- To tick one, place an x between two square brackets e.g. [x] -->

Reviewer: ```suggestion
<!-- To tick a box, place an `x` between two square brackets e.g. [x] -->

```

## Requirements
- [ ] You have completed your website. <!-- This is not required if the domain you're registering is for emails. -->
- [ ] The website is reachable.  <!-- This is not required if you're using this domain for email, etc -->
- [ ] You're not using Vercel or Netlify.
- [ ] The CNAME record doesn't contain `https://` or `/`.  <!-- This is not required if you are not using a CNAME record. -->

> - [ ] There is sufficient information at the `owner` field.  <!-- You must have your email presented at `email` field. If you do NOT, then please include other social platform e.g. Discord, Twitter so we can contact you -->

Reviewer: ```suggestion
- [ ] There is sufficient information at the `owner` field.  <!-- You need to have your email presented at `email` field or you can specify another social platform e.g Discord or Twitter, so we can contact you. -->
```

> - [ ] The file contains information of your domain is in `domains/` folder and ends with `.json`.

Reviewer: ```suggestion
- [ ] The file is in the `domains` folder and is in the JSON format
```

<!-- To make our job easier, please spend time to review your application before submitting. -->
<!-- To tick a box, place an `x` between two square brackets e.g. [x] -->

## Requirements
- [ ] You have completed your website. <!-- This is not required if the domain you're registering is for emails. -->

> - [ ] The website is reachable.  <!-- This is not required if you're using this domain for email, etc -->

Reviewer: ```suggestion
- [ ] The website is reachable.  <!-- This is not required if the domain you're registering is for emails. -->
```

- [ ] You're not using Vercel or Netlify.
- [ ] The CNAME record doesn't contain `https://` or `/`.  <!-- This is not required if you are not using a CNAME record. -->

> - [ ] There is sufficient information at the `owner` field.  <!-- You need to have your email presented at `email` field or you can specify another social platform e.g Discord or Twitter, so we can contact you. -->

Reviewer: ```suggestion
- [ ] There is sufficient information at the `owner` field.  <!-- You need to have your email presented at `email` field or you can specify another social platform (e.g. Discord or Twitter) so we can contact you. -->
```

> - [ ] The file is in the `domains` folder and is in the JSON format.

Reviewer: ```suggestion
- [ ] The file is in the `domains` folder and is in the JSON format.

```
